lib/foods.js

```diff
@@ -8,7 +8,9 @@ $(document).ready(function() {
   $('.food-options').on('click', '.food-delete', () => {
     handleDeletingFood();
   });
-
+  $('.food-options').on('click', '.food-name, .food-calories', () => {
```

Comment:
I thought that pushing the 'enter' key worked automatically in forms. So maybe if the input field was wrapped in a form?